app/assets/javascripts/application.js

```diff
@@ -7,4 +7,5 @@
 //= require modules/feeds.js
 //= require modules/toggle-attribute
 //= require components/accordion
+//= require modules/coronavirus_landing_page
```

Comment:
minor, we tend to use dash-separated filenames for JavaScript, including Collections